charts/apisix-ingress-controller/templates/deployment.yaml

```diff
@@ -103,6 +103,13 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.name
+          {{- if .Values.config.apisix.existingSecret }}
+          - name: APISIX_ADMIN_KEY
```

Comment:
@tao12345666333 i guess your proposal is to try to use `DEFAULT_CLUSTER_ADMIN_KEY` environment variables instead of the `APISIX_ADMIN_KEY`?

Reply:
yes

Reply:
in other case i used exactly the same approach as in your tests

Reply:
done